### DISCUSSION TITLE
How to add a Github pages contact form

### DISCUSSION THREAD

**Discussion by Irishgeoff12** [community/community#67047] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

This is a handy tutorial if you would like to add  a [Github pages contact form](https://fabform.io/a/create-a-static-website-with-contact-form-on-github-pages)

**Comment by queenofcorgis** [comment]
Hi @Irishgeoff12 , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
